src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Adding Layer 2's
+description: The policy we use when adding layer 2's to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2's {#adding-layer-2}
+
+Anyone is free to suggest new layer 2's on ethereum.org.
+
+We currently list them on:
+
+- [ethereum.org/layer-2](/layer-2/)
```

Comment:
We shouldn't add this page until it exists. Currently we list L2s on:
- https://ethereum.org/en/developers/docs/scaling/optimistic-rollups/
- https://ethereum.org/en/developers/docs/scaling/zk-rollups/
- ...any others?

Reply:
For sure, wasn't quite sure when the plan was to put this out, but ill update this to current pages and we can come back after layer-2 page is out.